docs/2/technical/quick-start/data-integration.md

```diff
@@ -34,54 +34,91 @@ If you are a Microshare® reseller, you can send the data to Microshare® and on
 
 <br>
 
-## 2. From Somewhere to Microshare
+## 2. Ingesting Data into Microshare®
 ---------------------------------------
 
-To integrate the data into Microshare® there are three methods. 
+To integrate the data into Microshare® there are three methods: 
 
-First, you push the data to Microshare®. 
+### A) Network Server to push data.
 For an IoT Server network you push the data on a rectype as shown on this page: 
 ##### > [Data Ingestion](/docs/2/technical/microshare-platform-advanced/data-ingestion/)
-Each time a new data is created it is sent to Microshare® and if the data formatting tools are in place the data will be automatically processed and improved.
-
-The second method is to use robots which are further elaborated on this page:
-##### > [Data Ingestion By Robots](/docs/2/technical/microshare-platform-advanced/data-ingestion-by-robots/)
-With this you will be able to create Websockets or even pull data at regular time intervals.
 
-Finally you can use our simple API request to write into Microshare® with an API.
+### B) HTTP POST to push data.
+The API supports RESTful POST operations to inject data into the system as JSON documents. 
 ##### > [APIs](/docs/2/technical/api/quick-start/)
 
+### C) Robots polling API endpoints to pull data.
+Robots may run Javascript that can be used to pull in data via APIs at regular time intervals.
+##### > [Data Ingestion By Robots](/docs/2/technical/microshare-platform-advanced/data-ingestion-by-robots/)
```

Comment:
You removed this page, should I link the robot guide?

Reply:
*data ingestion (without robots)

Reply:
that is what I would have done.